CustomerAccount.sol

```diff
@@ -4,9 +4,11 @@ pragma solidity ^0.5.0;
 contract CustomerAccount{
     address owner;
     bool isNewAccount;
-    uint public accountBalance;
```

Comment:
This is a conflict with main.

Reply:
Public access control for this state variable is correct so that it can searched by others.

Reply:
ok.